supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -1047,14 +1069,25 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 [discrete]
 [[rpm]]
 ==== image:images/yes.png[yes] RPM (noun)
-*Description*: _RPM_ is the recursive initialism for the _RPM Package Manager_. RPM manages files in the RPM format, known as RPM packages. RPM packages are known informally as `rpm` files, but this informal usage is not used in Red Hat documentation to avoid confusion with the command name. Files in RPM format are referred to as "RPM packages".
+*Description*: _RPM_ is the recursive initialism for the _RPM Package Manager_. RPM manages files in the RPM format. Those files are referred to as "RPM packages". Even though RPM packages are known informally as `rpm` files, do not use this informal term in Red Hat documentation to avoid confusion with the command name.
 
 *Use it*: yes
 
 *Incorrect forms*: rpm
 
 *See also*:
 
+[discrete]
+[[rpm-macro]]
+==== image:images/yes.png[yes] RPM macro (noun)
+*Description*: An _RPM macro_ is a straight text substitution that can be conditionally assigned based on the optional evaluation of a statement when certain built-in functionality is used. RPM macros help to make package maintenance simpler and to automate certain repetitive tasks or patterns that are common across all types of packages. An RPM macro is a fragment of text with an assigned name. Whenever the name is used, it is replaced by the expanded body of the macro. This process is called "macro expansion" and can range from a static text substitution to arbitrary code running with side effects.
```

Comment:
This is a very long definition. Can we cut the last two sentences ("An RPM macro is a fragment... running with side effects")?

Reply:
It  is actually 3 sentences :-) discussed with SME and cutting those sentences.